src/PayloadParser.h

```diff
@@ -394,7 +394,7 @@ namespace snowcrash {
                                    const ParseResultRef<Payload>& out) {
 
             const char* regex;
-            mdp::ByteBuffer mediaType;
+	    mdp::ByteBuffer mediaType;
```

Comment:
Actually, there is one tab and four spaces at the start of this line. Changing the tab back to spaces should resolve.